web/src/components/core/ContextualActions.jsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState } from 'react';
+import { Button, Dropdown, DropdownItem, DropdownGroup } from '@patternfly/react-core';
+import { Icon, ContextualActions as ContextualActionsSlot } from "~/components/layout";
+
+const Toggler = ({ onClick }) => {
+  return (
+    <Button onClick={onClick} variant="plain">
+      <Icon name="expand_more" />
```

Comment:
I actually like the "+" icon, which for me has a lot of sense (_more page actions_). But since I thought it might be a bit controversial... I opted for using a "dropdown arrow" like one. Anyway, see how they look side by side

| "Expand more" icon | "Add" icon |
|-|-|
| ![Storage page using "expand more" icon](https://user-images.githubusercontent.com/1691872/233946775-5315d902-fabb-490e-8146-f18d3ce56e4f.png) | ![Storage page using "add" icon](https://user-images.githubusercontent.com/1691872/233946799-20646ce0-c86c-4812-84f7-8e5e19406e8c.png) |

Either way, there are plenty of them at https://fonts.google.com/icons. Have a look to see if you find one that fits better.

Reply:
I like both, and I see the down arrow more generic. I would it for now.

Reply:
As said, I also like the arrow. But I am asking myself if the icon is dicoverable enough. At first sight it looks to me like a button for minimizing or something similar (maybe it is my fault as user). Moreover, how can we refer to it? I mean, in the devices selector popup we have a sentence for suggesting the user to configure more devices if needed. Do we need to keep that tip? If so, how?

Reply:
BTW, how do you see to add a button? E.g., "More options V", with good styles to integrate it nicely with the sidebar.

Reply:
I do not think it is needed such verbosity. But if you want, go ahead.

Reply:
Maybe I'd use the tooltip here.